src/MODULE.tools

```diff
@@ -10,7 +10,7 @@ bazel_dep(name = "buildozer", version = "7.1.2")
 bazel_dep(name = "platforms", version = "0.0.10")
 bazel_dep(name = "zlib", version = "1.3.1.bcr.3")
 
-cc_configure = use_extension("//tools/cpp:cc_configure.bzl", "cc_configure_extension")
+cc_configure = use_extension("@rules_cc//cc:extensions.bzl", "cc_configure_extension")
 use_repo(cc_configure, "local_config_cc", "local_config_cc_toolchains")
 register_toolchains("@local_config_cc_toolchains//:all")
```

Comment:
Yes, this is also the situation previously. rules_cc register the same toolchain from `@bazel_tools` as well. I'm a bit worried about if the BFS order of toolchain registration would cause any problem if we purely rely on rules_cc, but maybe it's fine?

Reply:
BFS order could change, but at least there would now be a way to explicitly order rules_cc and apple_support toolchains. I think that's a plus.

Reply:
SG, then I'll just remove this from MODULE.tools.